src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace Our.Umbraco.Ditto
 {
+    using global::Umbraco.Core;
```

Comment:
We can place the `using` namespaces outside the main namespace. Those left inside are older code. The `global::` part felt unnecessary.